modules/openy_features/openy_node/modules/openy_node_news/openy_node_news.info.yml

```diff
@@ -0,0 +1,28 @@
+name: 'OpenY Node News (experimental)'
+description: 'OpenY news features.'
+type: module
+core: 8.x
+dependencies:
```

Comment:
this module should not have a dependency to openY modules like `openy_loc_branch`, `openy_loc_camp`, `openy_txnm_color`, `openy_txnm_blog_category`